setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
Hey, you have a good point there!  Sounds like you have more experience with Python then me.. lol :)   Let me change my PR as you suggest.